packages/platform/platform-http/src/common/services/PlatformHandler.ts

```
@@ -26,14 +26,13 @@ export class PlatformHandler {
   protected responseFilter = inject(PlatformResponseFilter);
   protected platformParams = inject(PlatformParams);
   protected platformExceptions = inject(PlatformExceptions);
-  protected platformMiddlewaresChain = inject(PlatformMiddlewaresChain);
   protected platformRouters = inject(PlatformRouters);
 
   constructor() {
     // configure the router module
     this.platformRouters.hooks
       .on("alterEndpointHandlers", (handlers: AlterEndpointHandlersArg, operationRoute: JsonOperationRoute) => {
-        handlers = this.platformMiddlewaresChain.get(handlers, operationRoute);
+        handlers = $alter("$alterEndpointHandlers", handlers, [operationRoute]);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)